app/views/common/DescriptionTemplate.tsx

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import {
+  ImageBackground,
+  StyleSheet,
+  TouchableOpacity,
+  TextStyle,
+  View,
+  ScrollView,
+  ImageSourcePropType,
+  ViewStyle,
+} from 'react-native';
+import { SvgXml } from 'react-native-svg';
+
+import { useStatusBarEffect } from '../../navigation';
+import { Typography } from '../../components/Typography';
+
+import {
+  Buttons,
+  Colors,
+  Layout,
+  Spacing,
+  Iconography,
+  Typography as TypographyStyles,
+} from '../../styles';
+
+type DescriptionTemplateProps = {
+  iconXml: string;
+  header: string;
+  headerStyle?: TextStyle;
+  body: string;
+  bodyStyle?: TextStyle;
+  primaryButtonLabel: string;
+  primaryButtonContainerStyle?: ViewStyle;
+  primaryButtonTextStyle?: TextStyle;
+  primaryButtonOnPress: () => void;
+  secondaryButtonLabel?: string;
+  secondaryButtonContainerStyle?: ViewStyle;
+  secondaryButtonTextStyle?: TextStyle;
+  secondaryButtonOnPress?: () => void;
+  backgroundImage: ImageSourcePropType;
+  backgroundStyle?: ViewStyle;
+  invertIcon?: boolean;
+};
+
+const DescriptionTemplate = ({
+  iconXml,
+  header,
+  headerStyle,
+  body,
+  bodyStyle,
+  primaryButtonLabel,
+  primaryButtonContainerStyle,
+  primaryButtonTextStyle,
+  primaryButtonOnPress,
+  secondaryButtonLabel,
+  secondaryButtonContainerStyle,
+  secondaryButtonTextStyle,
+  secondaryButtonOnPress,
+  backgroundImage,
+  backgroundStyle,
+  invertIcon,
+}: DescriptionTemplateProps): JSX.Element => {
+  useStatusBarEffect('dark-content');
+
+  const iconStyle = invertIcon ? styles.goldIcon : styles.blueIcon;
+
+  const primaryButtonTextStyles = {
```

Comment:
We have to do this instead of inlining `style={[styles.primaryButtonText, primaryButtonTextStyle}]` because the `Typography` component's `style` prop expects a `TextStyle`.